app/i18n/he.json

```diff
@@ -172,14 +174,14 @@
     "month": "חודש",
     "monthlyPrice": "%{price}/חודש",
     "yearlyPrice": "%{price}/שנה",
-    "liveActivity": "עדכונים בזמן אמת",
+    "liveActivity": "בטר רייל לייב",
```

Comment:
Let's stop with the Hebresh, I promise you Avshalom Kor won't judge 😇

Reply:
doesn't look good in english imo
<img src="https://user-images.githubusercontent.com/13344923/241484773-a1df65d3-e4ba-4f7b-9a1b-2ac41a68be40.png" alt="Simulator Screenshot - iPhone 12 mini - 2023-05-28 at 09 32 16" width="260" />

Reply:
It looks good IMO!!